e2e/runTest.sh

```diff
@@ -15,6 +15,8 @@ TEST_TARGET="$1"
 # second argument (optional) of the script should be a path to the directory with tested project (relative to the project root)
 TEST_PROJECT_PATH="$2"
 
+export env = "foo1 : foo2"
```

Comment:
in bash the syntax is
```
export NAME=VALUE
```

also, we'd like to make it a bit more generic - these envs could be read from the arguments, and then we can set them only for the specific test in BUIOLD file

also it's possible to set them only for the single bash call:
```
NAME=VALUE ./bash_srcipt.sh
```

it's cleaner

Reply:
```
foo1=invalid_val1 foo2=invalid_val2 foo3=val3 foo4=val4 bazel run "$TEST_TARGET"
```